### ISSUE TITLE
[BUG]: Issues happening in translations

### ISSUE THREAD

**Issue by IlDucci** [PCSX2/pcsx2#9062]
### Describe the Bug

This is a new look made at the current state of PCSX2 localization and the issues that have appeared during the testing of the first PR that had languages included. It might affect both English texts and non-English. Might also be a master thread of sorts.

 - Clear Bindings question: "Are you sure you want to clear all bindings for this controller?" should read "for this USB device?"
 - Settings->Graphics->Post-Processing: add a colon to "Saturation".
 - Main menu -> Toolbar: Decouple the toolbar strings from the rest so localized versions can try to shorten/abbreviate the toolbar buttons. (akin to what Dolphin does). Case in point:
![imagen](https://github.com/PCSX2/pcsx2/assets/5641681/4c722e3c-4654-475d-b0c9-13983fa6888c)
 - Menu bar -> Settings: Missing hotkeys for a number of options (UI, Game List, Emulation...)
 - Menu bar -> Overall missing hotkeys for a number of options
 - Some "Form" texts that should not be translatable have popped back up:
   - ../Settings/GameCheatSettingsWidget.ui:14
   - ../Settings/GamePatchDetailsWidget.ui:20
   - ../Settings/PatchDetailsWidget.ui:14
   - ../Settings/GamePatchSettingsWidget.ui:14
 - Is it possible to remove from translations some of the most unwanted HTML codes? Like here: `&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;`
 - Settings->Interface-> Maybe have a mandate that all non-English languages translate "Language:" as "Language [in the destination language] (Language) [in English]:"?
 - Settings -> Graphics -> Maybe decouple all the "None"/Auto" strings so genders/numbers can be customizable depending on the option?
 - Controllers -> Hotkeys -> The "&" that links multiple combinations should be translatable.
 - Controllers -> USB -> Microphones (Singstar, Logitech, etc): "Not Connected" and "Default Input Device" are not translatable.
 - RetroAchievements login window: The window size is hardcoded, so whenever a longer string appears, the window seems to be jumping around (reported on Discord).
 - Game Properties -> Rendering -> Display: The Screenshot Size area is fudged/not fit for translation, and also, many options that have short boxes are broken (trying to display "Default: (value)").
![imagen](https://github.com/PCSX2/pcsx2/assets/5641681/8634f014-0c25-4ece-b8b9-4afc7248112b)
 - Game Properties -> Rendering -> Post-Processing: more of the same.
![imagen](https://github.com/PCSX2/pcsx2/assets/5641681/3b464bf3-9d1b-4665-9a20-c170328fc7e9)
 - Controls -> Hotkeys -> The "&" that appears whenever a Hotkey has multiple entries should be translatable.
![imagen](https://github.com/PCSX2/pcsx2/assets/5641681/1394260f-e4c0-44ce-8429-2e97f830030a)
 - 

### Reproduction Steps

Go to each menu in a non-English language using stenzek's PR artifacts: https://github.com/PCSX2/pcsx2/pull/9054

### Expected Behavior

Cleaner translated UI.

### PCSX2 Revision

v1.7.4622 (lie for now to get the issue working)

### Operating System

Windows 10 (64bit)

**Comment by stenzek** [comment]
> Controls -> Hotkeys -> The "&" that appears whenever a Hotkey has multiple entries should be translatable.

This one's a WONTFIX. The & is a control character in how the binding is parsed, and the binding itself is not translatable.

**Comment by IlDucci** [comment]
List updated, focusing on the Settings window. I'm leaving things at the Emulation section (included) for now.

**Comment by IlDucci** [comment]
Updated the list with the comments mentioned in https://github.com/PCSX2/pcsx2/issues/9122 .

**Comment by xujibbs** [comment]
One more to be added in the list
Settings-Graphics-Rendering-Anisotrooic filtering-Strings "off (Default)
Settings-Graphics-Rendering-Internal resolution-string "Native (PS2) (Default)"
In msgctxt "SetupWizardDialog|", When I try to translate "&Next" and "&Finish", "&Next" can show translation once when I switch the language but return to English when I click "Next". I also can't make "&Finish" translatable.

**Comment by stenzek** [comment]
> Settings -> Emulation -> System Settings: Decouple the "Disabled" string. So far I only see two copies (EE Cycle Skipping and Affinity Control), but there seems to be four?
> Settings -> Graphics -> Maybe decouple all the "None"/Auto" strings so genders/numbers can be customizable depending on the option?

This one's a nope. The noop macro doesn't have disambiguation.

**Comment by IlDucci** [comment]
> > Settings ->Interface -> Maybe change the "Language:" string so all non-English languages display instead "Language [in the destination language] (Language) [in English]:"?
> 
> Also a won't fix - that would **massive** increase the number of items in the language list, as not all languages follow the same sentence form.

Just to be sure I did not misexplain, this is the intended effect: "Idioma (Language):". Just the setting name, not the available languages.

**Comment by IlDucci** [comment]
Revision updated: I have finished going through the Settings and Controller windows. Still have to check all the other windows.

**Comment by stenzek** [comment]
> Just to be sure I did not misexplain, this is the intended effect: "Idioma (Language):". Just the setting name, not the available languages.

Not really sure what you mean by this. The setting name is part of the .ui file, and should already be translatable.

You can add additional context/information on crowdin.

**Comment by nikivarvar** [comment]
In Rendering -> Post-Processing, the Constant Adaptive Sharpening doesn't seem to show up anywhere, where the string have been modified in any way.

**Comment by kamfretoz** [comment]
The "Scan Recursively" column on the Game List setting needs to be made wider to accomodate with translation strings in cases where they are longer than the source string

![image](https://github.com/PCSX2/pcsx2/assets/14798312/e665b798-d62e-4752-b0e3-89fc484fc576)

**Comment by bigol83** [comment]
Game List view: hours,minutes and the "never" words to indicate how much time we played and when the game has never been played yet, cannot be translated. Searched strings on Crowding and confirmed we cannot translate it yet

**Comment by piorrro33** [comment]
Fairly sure "quadrafonic" is supposed to be "quadraphonic".

**Comment by bigol83** [comment]
Little update: now the game list playtime words such as "seconds", "hours", "minutes", "never", "today", "yesterday" can be translated on Crowdin. 

But it seems date format is still based on US, there is no option to change it into another format. This can be confusing for European users. Dates such as 08/05/2023 mean May 8th in Europe, while they mean August 5th in US format.

**Comment by stenzek** [comment]
It should use whatever the locale of your system is, for date formatting.

**Comment by bigol83** [comment]
Checked my local date formatting and it uses the EU one as it should.

**Comment by IlDucci** [comment]
I can confirm the locale settings are not being used by PCSX2 in a Windows PC with Spanish (Spain) settings.
![imagen](https://github.com/PCSX2/pcsx2/assets/5641681/372cc3a8-01a8-478e-98f2-afdd2a69d6d3)

**Comment by IlDucci** [comment]
Translations meta thread updated with all the reports, plus a bunch of untranslated strings that I just found in Big Picture. These strings might not be the only ones, so if any of the reporters reads this, please do some research work as well.

**Comment by Hazer-123** [comment]
There is a small typo in the following string below. "Calculate", not "Calclate".
![1694878891913](https://github.com/PCSX2/pcsx2/assets/122793818/ec99f9ca-6a95-4fd9-9a55-fe2d7c47f28d)

**Comment by refractionpcsx2** [comment]
The Interlace dropdown says `None (Interlaced, also used by Progressive)` which is incorrect, it should probably just say `None (No Interlacing)` I don't think it needs the progressive addition here.

**Comment by refractionpcsx2** [comment]
It is?

**Comment by RedDevilus** [comment]
> It is?

What about if the game doesn't give/have progressive signals? I thought that would just be Interlaced unless I understood it wrong.

**Comment by refractionpcsx2** [comment]
if the game doesn't have progressive signals it'll look shaky. I'm saying "No Interlacing" so it doesn't do the interlacing, so if it looks broken, that's your own fault, what do they want from us? :P

**Comment by refractionpcsx2** [comment]
I guess maybe it should be `No Deinterlacing` since that's the process. Is that clearer for you?

**Comment by piorrro33** [comment]
I believe "Expansion" should be "Upmixing". "Upmixing" is a much more common audio term.

Google searches:
https://www.google.com/search?q=audio+expansion
https://www.google.com/search?q=audio+upmixing

**Comment by RedDevilus** [comment]
> I believe "Expansion" should be "Upmixing". "Upmixing" is a much more common audio term.
> 
> Google searches:
> https://www.google.com/search?q=audio+expansion
> https://www.google.com/search?q=audio+upmixing

If I remember correctly Expansion was used because it's virtual emulation of surround sound so faking it more-so but a fair remark.

Not sure what to do with it so I'll leave it to someone else at the moment.

**Comment by IlDucci** [comment]
List updated with a full, comprehensive sublist for Big Picture. I am also going to attempt to make a few strings translatable.

**Comment by HarmfulBreeze** [comment]
This string should be "CPU Sprite Render Size", or other similar strings should be adjusted accordingly
https://crowdin.com/proofread/pcsx2-emulator/15/en-fr#5143
https://github.com/PCSX2/pcsx2/blob/ec53e8949a4affafb5315efd5af8f52c9d3a034f/pcsx2-qt/Settings/GraphicsSettingsWidget.cpp#L512

**Comment by IlDucci** [comment]
> This string should be "CPU Sprite Render Size", or other similar strings should be adjusted accordingly https://crowdin.com/proofread/pcsx2-emulator/15/en-fr#5143
> 
> https://github.com/PCSX2/pcsx2/blob/ec53e8949a4affafb5315efd5af8f52c9d3a034f/pcsx2-qt/Settings/GraphicsSettingsWidget.cpp#L512

Fixed in https://github.com/PCSX2/pcsx2/pull/10363 .

**Comment by xujibbs** [comment]
It might create a new topic to collect translation issue. Pcsx2 changes too much.

**Comment by IlDucci** [comment]
It's not changing that much. The OP could use some spring cleaning, though. I'll get with it.

**Comment by IlDucci** [comment]
Meta thread for untranslatable strings and other translation-related issues updated to the current state of version 2.0. Only a tiny amount of issues were solved since May.

**Comment by bigol83** [comment]
The word "Version" shown on About dialogue screen in Big Picture mode cannot be translated. 

![Cattura3](https://github.com/user-attachments/assets/7c8ead43-620f-4333-8dc4-3df3038c3a6b)

**Comment by IlDucci** [comment]
> The word "Version" shown on About dialogue screen in Big Picture mode cannot be translated.

Added to the list, thanks!

**Comment by IlDucci** [comment]
Updating the Meta thread with the latest fixes provided in https://github.com/PCSX2/pcsx2/commit/2fa85c56c211f5579dba4f2280b39fb815c1fdb7 .

**Comment by IlDucci** [comment]
New menu, new issues. Updating the Meta thread with all issues that stem from the new Debugger windows.

**Comment by TheLastRar** [comment]
> * Controls -> Hotkeys -> The "&" that appears whenever a Hotkey has multiple entries should be translatable.

In the SDL3 update, this got switched to a "+" during changes in how binds are displayed
What was the reason that this should be translatable?

**Comment by xujibbs** [comment]
Pull request "GS: Misc adapter and vendor detection changes #11938" make the string "(default)" in Graphics(Adapter) untranslatable.

**Comment by lightningterror** [comment]
> Pull request "GS: Misc adapter and vendor detection changes [#11938](https://github.com/PCSX2/pcsx2/pull/11938)" make the string "(default)" in Graphics(Adapter) untranslatable.

That's intended.

**Comment by IlDucci** [comment]
> > Pull request "GS: Misc adapter and vendor detection changes [#11938](https://github.com/PCSX2/pcsx2/pull/11938)" make the string "(default)" in Graphics(Adapter) untranslatable.
> 
> That's intended.

That's the only string of that dropdown box that should be translatable, IMHO. Adding it to the list.

**Comment by HarmfulBreeze** [comment]
https://github.com/PCSX2/pcsx2/blob/5a846296ca86f66bfe7af787b3c386e8355ae84d/pcsx2/ImGui/FullscreenUI.cpp#L5673

This string should be changed: it doesn't allow translating strings "Save" and "Backup Saves" and there might be languages where the word "Slot" would need to be placed between "Backup" and "Saves".

Therefore it should probably simply be split into two strings.

**Comment by IlDucci** [comment]
OP updated. The new OSD window has new untranslatable strings.

**Comment by RedDevilus** [comment]
<img width="1602" height="56" alt="Image" src="https://github.com/user-attachments/assets/e90de264-9d35-4250-b630-c22488dcd187" />

Stuff like the position of OSD is marked translateable, but PCSX2 doesn't make it available for translating instantly to translators. Might need the (manual) sync to update the English main template to push to the others from pcsx2-qt_en.ts file.

**Comment by IlDucci** [comment]
> Stuff like the position of OSD is marked translateable, but PCSX2 doesn't make it available for translating instantly to translators. Might need the (manual) sync to update the English main template to push to the others from pcsx2-qt_en.ts file.

That was the ticket, thanks. OP updated.

**Comment by chaoticgd** [comment]
> The decision to split "Go to in" and Disassembler/Memory View and "Add to" and Saved Addresses will not look nice at their fullest forms. I do not know if there's an intention of showing these strings in separate windows or boxes, but if that is not the case, I would suggest to retain the old strings with the full forms. Also, "Go to in" and "Add to" have been added to the translation file, but it's not being read at the moment.

The reason for this is that you can now open multiple memory views, and can also rename them. If you create a lot of them it creates an overflow submenu. Would it be better if "Go to in %1" and "Go to in..." (for the overflow submenu) were separate strings?

**Comment by IlDucci** [comment]
> The reason for this is that you can now open multiple memory views, and can also rename them. If you create a lot of them it creates an overflow submenu. Would it be better if "Go to in %1" and "Go to in..." (for the overflow submenu) were separate strings?

I just saw the overflow menu, I wasn't aware of it. I think that yes, both of those strings should be separate if possible, please.

**Comment by chaoticgd** [comment]
> > The reason for this is that you can now open multiple memory views, and can also rename them. If you create a lot of them it creates an overflow submenu. Would it be better if "Go to in %1" and "Go to in..." (for the overflow submenu) were separate strings?
> 
> I just saw the overflow menu, I wasn't aware of it. I think that yes, both of those strings should be separate if possible, please.

All the debugger stuff apart from the breakpoint dialog should be fixed by #13165.

**Comment by IlDucci** [comment]
> All the debugger stuff apart from the breakpoint dialog should be fixed by [#13165](https://github.com/PCSX2/pcsx2/pull/13165).

Thank you. There seems to be a regression in the Settings window, though.

I've done a full sweep of every issue reported here and it seems that the date format in QT's game list was also fixed at some point in time.

**Comment by chaoticgd** [comment]
> > All the debugger stuff apart from the breakpoint dialog should be fixed by [#13165](https://github.com/PCSX2/pcsx2/pull/13165).
> 
> Thank you. There seems to be a regression in the Settings window, though.
> 
> I've done a full sweep of every issue reported here and it seems that the date format in QT's game list was also fixed at some point in time.

I renamed a bunch of the graphics and debugger settings-related UI classes recently, that's probably the reason.

Edit: To be specific, I split up a bunch of the .ui files into smaller ones. The new ones are called "*Tab.ui" instead of "*Widget.ui".

**Comment by xujibbs** [comment]
Add 2untranslatable strings
“Supported Image Types (*.bmp *.gif *.jpg *.jpeg *.png *.webp)” in interface-game list background-Browse is untranslatable
“Audio Files (*.wav)” in achievements-Browse is untranslatable

**Comment by IlDucci** [comment]
> Add 2untranslatable strings “Supported Image Types (*.bmp *.gif *.jpg *.jpeg *.png _.webp)” in interface-game list background-Browse is untranslatable “Audio Files (_.wav)” in achievements-Browse is untranslatable

Added to the list, thank you.

New windows, new buggers: added new untranslatable strings to the ImGUI version of the Network and HDD Settings screen. For some reason, some windows now display a translated "OK" button, but "Cancel" is untranslatable?

**Comment by xujibbs** [comment]
New untranslated strings:
1) Settings - Network & HDD - Ethernet device: string "auto" in Dropdown list untranslatable
2) Big picture - Settings - Network & HDD - Ethernet device type: string "unset" in Dropdown list untranslatable
3) Big picture - Settings - Network & HDD - HDD imange selection - create new: all strings in Dropdown list untranslatable

**Comment by TheLastRar** [comment]
> New untranslated strings:
> 
>     1. Settings - Network & HDD - Ethernet device: string "auto" in Dropdown list untranslatable

Is this translatable in the Qt UI?

**Comment by xujibbs** [comment]
I can't translate it in the QT UI, even I didn't find where the string is.

**Comment by TheLastRar** [comment]
> I can't translate it in the QT UI, even I didn't find where the string is.

This should now be translatable under `DEV9` (as of https://github.com/PCSX2/pcsx2/pull/13625)

**Comment by kamfretoz** [comment]
> Settings -> Achievements -> Sound Effects -> Browse... -> Accepted file types are untranslatable.
> Settings -> Interface -> Appearance -> Game List Background -> Browse... -> Accepted file types are untranslatable.

Should be taken care of once #13677 is merged

**Comment by IlDucci** [comment]
Meta translations thread updated with a number of issues marked as fixed.

**Comment by IlDucci** [comment]
Meta translations thread updated. Some issues are fixed, some are new.

**Comment by HarmfulBreeze** [comment]
The " (X seconds)" part can't be translated: https://github.com/PCSX2/pcsx2/blob/4cdda8d99ba12090bde7cfc58bb2bc410af75499/pcsx2/GS/Renderers/Common/GSRenderer.cpp#L968-L969

**Comment by IlDucci** [comment]
Added a new issue with the breakpoint window.
